# Statistics Case Study II Solutions

## Problem Statement 1:

In each of the following situations, state whether it is a correctly stated hypothesis testing problem and why?

#### 1. 𝐻0: 𝜇 = 25, 𝐻1: 𝜇 ≠ 25

<b>Solution: </b>

The null hypothesis contains equality. 
As a result the above statement is CORRECT.

#### 2. 𝐻0: 𝜎 > 10, 𝐻1: 𝜎 = 10

<b>Solution: </b>

Null hypothesis should contain equality. In the above case it does not. 
Hence the above statement is INCORRECT.

#### 3. 𝐻0: 𝑥̅ = 50, 𝐻1: 𝑥̅ ≠ 50

<b>Solution: </b>

Assuming that we are going to use the Z score statistic, hpothesis should always be applied for a population and not a sample. 
The above is for a sample.
   
Hence the above statement is INCORRECT. 

#### 4. 𝐻0: 𝑝 = 0.1, 𝐻1: 𝑝 = 0.5

<b>Solution: </b>

THe alternate hypothesis should state the opposite of the null hyopethsis.
In the above case the alternate hypothesis should have been 𝐻1: 𝑝 > 0.1

So the above statement is INCORRECT.

#### 5. 𝐻0: 𝑠 = 30, 𝐻1: 𝑠 > 30

<b>Solution: </b>

Here again assuming that we are going to use thr Z score statistic, standard deviation for the population needs to be used.
Here s is the standard deviation for a sample.

Hence the above statement is INCORRECT.

## Problem Statement 2:

The college bookstore tells prospective students that the average cost of its textbooks is Rs. 52 with a standard deviation of Rs. 4.50. A group of smart statistics students thinks that the average cost is higher. To test the bookstore’s claim against their alternative, the students will select a random sample of size 100. Assume that the mean from their random sample is Rs. 52.80. Perform a hypothesis test at the 5% level of significance and state your decision.

#### Solution:

![Problem_Statement_2_Solution](Pictures/Problem_Statement_2.jpg)

## Problem Statement 3:

A certain chemical pollutant in the Genesee River has been constant for several years with mean μ = 34 ppm (parts per million) and standard deviation σ = 8 ppm. A group of factory representatives whose companies discharge liquids into the river is now claiming that they have lowered the average with improved filtration devices. A group of environmentalists will test to see if this is true at the 1% level of significance. Assume \ that their sample of size 50 gives a mean of 32.5 ppm. Perform a hypothesis test at the 1% level of significance and state your decision.

#### Solution:

![Problem_Statement_3_Solution](Pictures/Problem_Statement_3.jpg)

## Problem Statement 4:

Based on population figures and other general information on the U.S. population, suppose it has been estimated that, on average, a family of four in the U.S. spends about
$1135 annually on dental expenditures. Suppose further that a regional dental association wants to test to determine if this figure is accurate for their area of country. To test this,
22 families of 4 are randomly selected from the population in that area of the country and a log is kept of the family’s dental expenditure for one year. The resulting data are given below. Assuming, that dental expenditure is normally distributed in the population, use the data and an alpha of 0.5 to test the dental association’s hypothesis.
1008, 812, 1117, 1323, 1308, 1415, 831, 1021, 1287, 851, 930, 730, 699,
872, 913, 944, 954, 987, 1695, 995, 1003, 994

#### Solution:

In [2]:
# Compute the mean of the samples that is provided
import statistics as st
import math

sample = [1008, 812, 1117, 1323, 1308, 1415, 831, 1021, 1287, 851, 930, 730, 699, 872, 913, 944, 954, 987, 1695, 995, 1003, 994]
sample_mean = st.mean(sample)
sample_mean

1031.3181818181818

In [6]:
# Compute the standaard deviation for the samples that are provided
sample_std_dev = st.stdev(sample)

In [8]:
t = (sample_mean - 1135) / (sample_std_dev / math.sqrt(22)); t

-2.023137479931484

![Problem_Statement_4_Solution](Pictures/Problem_Statement_4.jpg)

## Problem Statement 5:

In a report prepared by the Economic Research Department of a major bank the
Department manager maintains that the average annual family income on Metropolis is
$48,432. 

What do you conclude about the validity of the report if a random sample of 400
families shows and average income of $48,574 with a standard deviation of 2000?

#### Solution:

In [9]:
sample_mean = 48574
population_mean = 48432
sample_std_dev = 2000
n = 400

t = (sample_mean - population_mean) / (sample_std_dev / math.sqrt(400))
t

1.42

![Problem_Statement_5_Solution](Pictures/Problem_Statement_5.jpg)

## Problem Statement 6:

Suppose that in past years the average price per square foot for warehouses in the United
States has been $32.28. 

A national real estate investor wants to determine whether that
figure has changed now. The investor hires a researcher who randomly samples 19
warehouses that are for sale across the United States and finds that the mean price per
square foot is $31.67, with a standard deviation of $1.29. assume that the prices of
warehouse footage are normally distributed in population. If the researcher uses a 5%
level of significance, what statistical conclusion can be reached? What are the
hypotheses?

#### Solution:

##### Given:

population mean = 32.28

n = 19

sample mean = 31.67

sample std dev = 1.29

alpha = 0.05

In [10]:
(31.67-32.28)/(1.29/math.sqrt(19))

-2.06118477175179

![Problem_Statement_6_Solution](Pictures/Problem_Statement_6.jpg)

## Problem Statement 7:

![PS_7.png](PS_7.png)

#### Solution:

![Problem_Statement_7a_Solution](Pictures/Problem_Statement_7a1.jpg)

![Problem_Statement_7a_Solution](Pictures/Problem_Statement_7a2.jpg)

![Problem_Statement_7b_Solution](Pictures/Problem_Statement_7b1.jpg)

![Problem_Statement_7b_Solution](Pictures/Problem_Statement_7b2.jpg)

![Problem_Statement_7c_Solution](Pictures/Problem_Statement_7c1.jpg)

![Problem_Statement_7c_Solution](Pictures/Problem_Statement_7c2.jpg)

![Problem_Statement_7d_Solution](Pictures/Problem_Statement_7d1.jpg)

![Problem_Statement_7d_Solution](Pictures/Problem_Statement_7d2.jpg)

![Problem_Statement_7d_Solution](Pictures/Problem_Statement_7d2.jpg)

![Problem_Statement_7_Table](Pictures/Problem_Statement_7.jpg)

## Problem Statement 8:

Find the t-score for a sample size of 16 taken from a population with mean 10 when the
sample mean is 12 and the sample standard deviation is 1.5

#### Solution:

In [11]:
Popultion_mean = 10
sample_size = 16
sample_mean = 12
sameple_sd = 1.5

t_score = (sample_mean - Popultion_mean)/ (sameple_sd/math.sqrt(sample_size))
t_score

5.333333333333333

## Problem Statement 9:

Find the t-score below which we can expect 99% of sample means will fall if samples of
size 16 are taken from a normally distributed population.

![Problem_Statement_9_Solution](Pictures/Problem_Statement_9.jpg)

## Problem Statement 10:

If a random sample of size 25 drawn from a normal population gives a mean of 60 and a
standard deviation of 4, find the range of t-scores where we can expect to find the
middle 95% of all sample means. Compute the probability that 𝑃(−𝑡0.05 < 𝑡 < 𝑡0.10).

![Problem_Statement_10_Solution](Pictures/Problem_Statement_10.jpg)

### Problem Statement 11:

![PS_11.png](Pictures/PS_11.png)

### Solution:

H0 : μ1 – μ2 = 0

H1 : μ1 – μ2 ≠ 0

![Problem_Statement_Z_Formula](Pictures/Z_Formula.jpg)

In [5]:
import math

In [6]:
Z = ((452 - 523) - 0)/math.sqrt((math.pow(212,2)/1200) + (math.pow(185,2)/800))
Z

-7.926428526759299

p-value: p(z < -7.926) = 0    &nbsp;&nbsp;&nbsp;   # As there is no correspoding value in the Z-Table so we are considering as 0

Assuming a significance level of 5%, the Z critial value +1.96, -1.96.
Clearly -7.926 is in the rejection region.


So this concludes that the null hypothesis (H0) can be rejected. The number of people travelling from Bangalore to Chennai is not same as the people travelling from Bangalore to Hosur.

### Problem Statement 12:

Is there evidence to conclude that the number of people preferring Duracell battery is
different from the number of people preferring Energizer battery, given the following:

Population 1: Duracell

n1 = 100

x1 = 308

s1 = 84

Population 2: Energizer

n2 = 100

x2 = 254

s2 = 67

### Solution:

H0 : μ1 – μ2 <= 0

H1 : μ1 – μ2 > 0

![Problem_Statement_Z_Formula](Pictures/Z_Formula.jpg)

In [9]:
Z = ((308 - 254) - 0)/math.sqrt((math.pow(84,2)/100) + (math.pow(67,2)/100))
Z

5.025702668336442

Assuming a significance level of 5%, the Z critial value +1.96, -1.96.
Clearly 5.026 is in the rejection region.

So this concludes that the null hypothesis (H0) can be rejected. 

### Problem Statement 13:

![PS_13.png](Pictures/PS_13.png)

### Solution:

H0 : μ1 – μ2 = 0

H1 : μ1 – μ2 ≠ 0

![t_Formula.png](Pictures/t_Formula.png)

In [4]:
t = ((0.317 - 0.21) - 0)/math.sqrt(((((14-1)*math.pow(0.12,2))+((9-1)*math.pow(0.11,2)))/(14+9-2))*((1/14)+(1/9)))
t

2.15355322387416

degrees of freedom = n1 + n2 - 2 = 14 + 9 - 2 = 21

Critical points of t for 2 tail tests.

t(0.05) = 2.08, t(0.10) = 1.721

So we can reject null hypothesis (H0) at the 10% level of significance.

## Problem Statement 14:

The manufacturers of compact disk players want to test whether a small price reduction
is enough to increase sales of their product. Is there evidence that the small price
reduction is enough to increase sales of compact disk players?

Population 1: Before reduction

n1 = 15

x1 = Rs. 6598

s1 = Rs. 844

Population 2: After reduction

n2 = 12

x2 = RS. 6870

s2 = Rs. 669

### Solution:

H0 : μ1 – μ2 <= 0

H1 : μ1 – μ2 > 0

![t_Formula.png](Pictures/t_Formula.png)

In [5]:
t = ((6598 - 6870) - 0)/math.sqrt(((((15-1)*math.pow(844,2))+((12-1)*math.pow(669,2)))/(15+12-2))*((1/15)+(1/12)))
t

-0.9098300343990461

Degrees of freedom = n1 + n2 - 2 = 15 + 12 -2 = 25

alpha = 10% = 0.10


t critical value at 0.10 is ±1.708
Clearly the computed value lies inside the non-rejection area.

So null hypothesis (H0) will not be rejected at the 10% level of significance.

## Problem Statement 15:

Comparisons of two population proportions when the hypothesized difference is zero.

Carry out a two-tailed test of the equality of banks’ share of the car loan market in 1980
and 1995.

Population 1: 1980

n1 = 1000

x1 = 53

𝑝̂ 1 = 0.53

Population 2: 1985

n2 = 100

x2 = 43

𝑝̂ 2= 0.53

### Solution:

H0 : p1 – p2 = 0

H1 : p1 – p2 ≠ 0

![Z_Formula_Probability.png](Pictures/Z_Formula_Probability.png)

In [6]:
P = (53 + 43)/(1000 + 100)
P

0.08727272727272728

In [7]:
Z = ((0.53 - 0.43) - 0)/math.sqrt(P * (1-P) * ((1/1000) + (1/100)))
Z

3.378266212559581

Z critical value at 95% level of significance = 1.645

Z critical value at 99% level of significance = 2.33

So this concludes, the null hypothesis (H0) can be rejected even at 99% level of significance.

## Problem Statement 16:

Carry out a one-tailed test to determine whether the population proportion of traveler’s
check buyers who buy at least $2500 in checks when sweepstakes prizes are offered as
at least 10% higher than the proportion of such buyers when no sweepstakes are on.

Population 1: With sweepstakes

n1 = 300

x1 = 120

𝑝̂ 1 = 0.40

Population 2: No sweepstakes

n2 = 700

x2 = 140

𝑝̂ 2= 0.20

### Solution:

H0 : p1 – p2 < 0.10

H1 : p1 – p2 ≥ 0.10

![Z_Formula_Probability_1.png](Pictures/Z_Formula_Probability_1.png)

In [8]:
Z = ((0.40 - 0.20) - 0.10)/math.sqrt(((0.4 * (1-0.4))/300) + ((0.2 * (1-0.2))/700))
Z

3.1180478223116177

alpha = 0.001

Z critical value at 0.001 level of significance is 3.09. So null hypothesis (H0) can be rejected at any level of significance.

## Problem Statement 17:

A die is thrown 132 times with the following results:

Number turned up: 1, 2, 3, 4, 5, 6

Frequency: 16, 20, 25, 14, 29, 28

Is the die unbiased? Consider the degrees of freedom as n − 1.

### Solution:

![Problem_Statement_17_Solution](Pictures/Problem_Statement_17.png)

## Problem Statement 18:

In a certain town, there are about one million eligible voters. A simple random sample of
10,000 eligible voters was chosen to study the relationship between gender and
participation in the last election. The results are summarized in the following 2X2 (read
two by two) contingency table:

![PS_18.png](PS_18.png)

We would want to check whether being a man or a woman (columns) is independent of
having
voted in the last election (rows). In other words, is “gender and voting independent”?

### Solution:

![Problem_Statement_18_Solution](Pictures/Problem_Statement_18.png)

## Problem Statement 19:

A sample of 100 voters are asked which of four candidates they would vote for in an
election. The number supporting each candidate is given below:

Higgins Reardon White Charlton

41      19      24      16

Do the data suggest that all candidates are equally popular? [Chi-Square = 14.96, with
3 df, 𝑝̂ < 0.05].

### Solution:

![Problem_Statement_19_Solution](Pictures/Problem_Statement_19.png)

## Problem Statement 20:

Children of three ages are asked to indicate their preference for three photographs of
adults. Do the data suggest that there is a significant relationship between age and
photograph preference? What is wrong with this study? [Chi-Square = 29.6, with 4 df: 𝑝̂ <
0.05].

![PS_20.png](PS_20.png)

### Solution:

![Problem_Statement_20_Solution](Pictures/Problem_Statement_20.png)

## Problem Statement 21:

A study of conformity using the Asch paradigm involved two conditions: one where one
confederate supported the true judgement, and another where no confederate gave the
correct response.

![PS_21.png](PS_21.png)

### Solution:

![Problem_Statement_21_Solution](Pictures/Problem_Statement_21.png)

## Problem Statement 22:

![PS_22.png](PS_22.png)

### Solution:

![Problem_Statement_22_Solution](Pictures/Problem_Statement_22.png)

## Problem Statement 23:

Each respondent in the Current Population Survey of March 1993 was classified as
employed, unemployed, or outside the labor force. The results for men in California age
35-44 can be cross-tabulated by marital status, as follows:

![PS_23.png](PS_23.png)

Men of different marital status seem to have different distributions of labor force status.
Or is this just chance variation? (you may assume the table results from a simple random
sample.)

### Solution:

![Problem_Statement_23_Solution](Pictures/Problem_Statement_23.png)